src/out/production/netappfiles-java-crr-sdk-sample/.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```

Comment:
I guess, this /src/out folder is the locally compiled output, correct? If that is the case, lets add this folder as well to the the main .gitignore file.